tests/contrib/actions/telegram/test_telegram_action.py

```diff
@@ -0,0 +1,31 @@
+import pytest
```

Comment:
We can create more tests for this action. Even if other actions doesn't have much tests.

Reply:
I agree with this, I just made the bare minimum, will add more tests before merging it.